src/IonBinaryReader.ts

```diff
@@ -176,6 +174,17 @@ export class BinaryReader implements Reader {
     return this._parser.decimalValue();
   }
 
+  bigIntValue() : JSBI {
```

Comment:
I forget, is this method going to be private? 

If this method is exposed does this mean that we now have JSBI in our external facing API? 
Is this the long term goal? If not should we wrap it with our own interface such that we can swap out implementations of BigInteger to shield us from client braking changes?

Reply:
> I forget, is this method going to be private?

It's meant to be public to give users access to ints greater than `(2^53)-1`.

> If this method is exposed does this mean that we now have JSBI in our external facing API? Is this the long term goal?

The long term goal is to move from JSBI to using [proper `BigInt`s](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt), which are already broadly (but not universally) supported by JS runtimes. JSBI gives us the ability to [add a babel build step](https://github.com/GoogleChromeLabs/babel-plugin-transform-jsbi-to-bigint) that will convert all of our JSBI static method uses to `BigInt` syntax when we're ready. Then customers can decide whether they want native `BigInt`s or the JSBI shim. See [the "Why?" section](https://github.com/GoogleChromeLabs/jsbi/#why) of the JSBI repo for more details.

> [...] should we wrap it with our own interface such that we can swap out implementations of BigInteger to shield us from client braking changes?

JSBI strictly adheres to [the BigInt TC39 proposal](https://tc39.es/proposal-bigint/), which moved to [stage 4 ("Final")](https://tc39.es/process-document/) last month, so breaking changes are very unlikely.